pkg/keyValuePairs.go

```diff
@@ -16,7 +16,7 @@ type KeyValuePairs []KeyValuePair
 
 // String returns the string representation of KeyValuePairs.
 func (k *KeyValuePairs) String() string {
-	return fmt.Sprintf("%#+v", k)
+	return ""
```

Comment:
This is a contentious one. The current implementation shows `(default &pkg.KeyValuePairs(nil))` as the default value when running with the `-h` flag.

I don't feel thats a very helpful default, especially for a non-developer. The `String` method seems to only be here to satisfy an interface so imo a zero value string is just as useful.